<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>651</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>520</width>
      <height>520</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(111, 0, 0);
gridline-color: rgb(255, 255, 255);</string>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Laksaman</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="imageresources.qrc">:/files/gametitle.png</pixmap>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>211</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="currentRound_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK TC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>ROUND X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="whoseMove_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>PLAYER X, MAKE YOUR MOVE!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="end_mv_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK SC</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
    background-color: rgb(48, 0, 0);
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;
}

QPushButton:hover {
    background-color: rgb(71, 1, 1);
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;
}

QPushButton:pressed {
    background-color: rgb(48, 0, 0);
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;     
}</string>
     </property>
     <property name="text">
      <string>END MOVE</string>
     </property>
    </widget>
    <widget class="QLabel" name="ore_label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Ore</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="stone_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wood_label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="wood_label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Wood</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="ore_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="food_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="add_building_btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
    background-color: #420000;
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #590000;
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;
}

QPushButton:pressed {
    background-color: #420000;
    color: rgb(255, 170, 0);
    border: 1px solid #d2ad1a; 
    border-radius: 6px;     
}</string>
     </property>
     <property name="text">
      <string>Build a property</string>
     </property>
    </widget>
    <widget class="QLabel" name="Resources_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK JP</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Resources</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="stone_label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Stone</string>
     </property>
    </widget>
    <widget class="QLabel" name="gold_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Gold</string>
     </property>
    </widget>
    <widget class="QLabel" name="food_label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Food</string>
     </property>
    </widget>
    <widget class="QLabel" name="gold_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="total_workers_lbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="properties_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK JP</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Properties</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>319</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="worker_mgmnt_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK JP</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Worker management</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="houses_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>HOUSES</string>
     </property>
    </widget>
    <widget class="QLabel" name="mines_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>390</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>MINES</string>
     </property>
    </widget>
    <widget class="QLabel" name="docks_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>390</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>DOCKS</string>
     </property>
    </widget>
    <widget class="QLabel" name="houses_amt_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK TC Bold</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="mines_amt_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK SC Bold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="docks_amt_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK SC Bold</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="tile_status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Mono CJK JP</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Tile status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="claimed_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>CLAIMED</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>TOTAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="claimed_tiles_lbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="total_tiles_lbl">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="claimed_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>WORKERS</string>
     </property>
    </widget>
    <widget class="QLabel" name="claimed_label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>OWNED  AVAILABLE</string>
     </property>
    </widget>
    <widget class="QLabel" name="claimed_label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>MINERS</string>
     </property>
    </widget>
    <widget class="QLabel" name="total_miners_lbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="available_workers_lbl">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="available_miners_lbl">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK JP</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="game_over_lbl">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans CJK JP Bold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 239, 6);</string>
    </property>
    <property name="text">
     <string>GAME OVER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>graphicsView</zorder>
   <zorder>title_label</zorder>
   <zorder>line_5</zorder>
   <zorder>game_over_lbl</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imageresources.qrc"/>
 </resources>
 <connections/>
</ui>
